library/Ivoz/Provider/Infrastructure/Persistence/Doctrine/Mapping/CallCsvScheduler.CallCsvSchedulerAbstract.orm.yml

```diff
@@ -0,0 +1,69 @@
+Ivoz\Provider\Domain\Model\CallCsvScheduler\CallCsvSchedulerAbstract:
+  type: mappedSuperclass
+  uniqueConstraints:
+    CallCsvScheduler_name_brand:
+      columns:
+        - name
+        - brandId
+    CallCsvScheduler_company:
+      columns:
+        - companyId
```

Comment:
Not sure about this constraints :thinking:

Reply:
What do you think about it @cruzccl?